### PR TITLE
Resolved review comments for native connectivity guide (overview.md and stc.md)

### PR DESCRIPTION
**Pull request by n-chaprana**
Signed-off-by: Nishant Chaprana <n.chaprana@samsung.com>

### CODE REVIEW

**Comment by mijin85cho** [comment]
FYI : This update is additional update of #162 which was reviewed by varunishg.

File: docs/application/native/guides/connectivity/stc.md [hunk 1]
```diff
@@ -3,17 +3,17 @@
 
 STC means Smart Traffic Control. It provides extensible packet-level control services, including per-app data usage, total data quota, and background app's data saving. STC Library provides APIs fulfilling below mentioned features for application development.
 
-This feature is supported in mobile and werable profile.
+This feature is supported in mobile, tv and wearable profile.
```
Comment (mijin85cho) [inline docs/application/native/guides/connectivity/stc.md]: Question : Does TV supported by native? I thought for now we have TV only in web profile.

Comment (n-chaprana) [reply docs/application/native/guides/connectivity/stc.md]: Yes, TV profile supports native API for STC F/W.

File: docs/application/native/guides/connectivity/stc.md [hunk 2]
```diff
@@ -49,9 +49,9 @@ To enable your application to use the STC API:
    stc_deinitialize();
    ```
```
Comment (mijin85cho) [inline docs/application/native/guides/connectivity/stc.md]: I think we need `<a name="retrieve_data_usage_system"></a>` here. Please check and add it if it is needed.

File: docs/application/native/guides/connectivity/stc.md [hunk 3]
```diff
@@ -137,9 +137,9 @@ To fetch statistics about total network data consumed by system:
    }
    ```
```
Comment (mijin85cho) [inline docs/application/native/guides/connectivity/stc.md]: I think we need `<a name="retrieve_data_usage_applications"></a>` here. Please check and add it if it is needed.

**Comment by mijin85cho** [comment]
FYI ) As it looks like only the link tag issue is left, I pre-applied the update on TD as https://developer.tizen.org/development/guides/native-application/connectivity-and-wireless/stc-smart-traffic-control. Please check the url and also update this PR to merge...

File: docs/application/native/guides/connectivity/stc.md (hunk 3 above)
Comment (n-chaprana) [reply docs/application/native/guides/connectivity/stc.md]: resolved, now link is working

File: docs/application/native/guides/connectivity/stc.md (hunk 2 above)
Comment (n-chaprana) [reply docs/application/native/guides/connectivity/stc.md]: resolved, now link is working

**Comment by mijin80lee** [comment]
TD Sync Completed.(cross-check)

**Pull request merged**